tests/zit/client.py

```diff
@@ -107,3 +103,13 @@ def create(self, resource: str, data: dict, **path_kwargs):
         response = self.request(url, method='POST', json=data)
         assert response.status_code == 201, response.json()
         return response.json()
+
+    def create_form(self, resource: str, data: dict, files: dict, **path_kwargs):
```

Comment:
this is not possible, our api guidelines state that we only support `application/json` as content-types :)

Reply:
deleted